### DISCUSSION TITLE
스피커/리스너 앱 디자인 관련 제안

### DISCUSSION THREAD

**Discussion by Jsueeee** [euphony-io/check-in-speaker#38] [General]
<img width="40%" src="https://user-images.githubusercontent.com/66951780/136704232-4bbfa87b-e0b1-43d6-9037-39d4e7596243.png"/> <img width="40%" src="https://user-images.githubusercontent.com/66951780/136704236-d1191c7e-647a-4b50-bbc9-b0e74b45bd9f.png"/>

동일한 기기에서 두 앱의 디자인 차이입니다.
디자인 간격 조절은 간단하니 맞추는 게 좋을 것 같은데 어떻게 생각하시나요?? @jopopcorn

**Comment by Jsueeee** [comment]
![image](https://user-images.githubusercontent.com/66951780/136704429-a95df158-5170-442f-a55b-4e9f6f139221.png)
 
버튼은 이 정도로 조금 더 두꺼운 게 좋을 것 같다는 의견입니다 :)

**Comment by jopopcorn** [comment]
앗 전에 요청해주신거 잊지 않고 [PR](https://github.com/euphony-io/wave-in-speaker/pull/36) 올려서 반영했습니다!
생각보다 미세하게 바뀌긴 했는데 매장용 앱에 맞게 마진값을 조정했습니다!! 다시 한번 확인해주실 수 있을까요?

**Reply by Jsueeee** [reply]
앗 몰랐네요 ㅋㅋㅋㅋ 넵 알겠습니다!

**Reply by jopopcorn** [reply]
생각해보니 전에 apk로 추출해서 보내줬던 거 같은데 apk로 보내드리면 될까요?!

**Reply by Jsueeee** [reply]
apk로 확인을 하고 있었긴 한데 레이아웃은 그냥 pr 올리신 거 봐도 될 것 같습니다!

**Reply by Jsueeee** [reply]
확인했는데 충분한 것 같습니다!

**Reply by yeonns** [reply]
혹시 최상단 액션바 색상도 @Jsueeee 님이 올려주신 스샷에서 다른데, 수정이 되었나요?
![image](https://user-images.githubusercontent.com/10149398/136704591-c668c59e-2f1c-4d8d-84c8-6fb6b5e4b951.png)

**Reply by Jsueeee** [reply]
저희가 '입장을 위한 음성 데이터'처럼 글자 크기를 좀 늘리는 게 나을 것 같네요!

**Reply by jopopcorn** [reply]
> 혹시 최상단 액션바 색상도 @Jsueeee 님이 올려주신 스샷에서 다른데, 수정이 되었나요? ![image](https://user-images.githubusercontent.com/10149398/136704591-c668c59e-2f1c-4d8d-84c8-6fb6b5e4b951.png)

앗 이부분은 수정 안되었습니다!! 상단바 색상 수정해서 올리겠습니다!

**Reply by jopopcorn** [reply]
> 저희가 '입장을 위한 음성 데이터'처럼 글자 크기를 좀 늘리는 게 나을 것 같네요!

아하 제가 수정하는 김에 리스너 앱에 맞게 글씨 크기를 맞춰도 될 것 같은데 그건 어떠실까요 ?_?

**Reply by Jsueeee** [reply]
> > 저희가 '입장을 위한 음성 데이터'처럼 글자 크기를 좀 늘리는 게 나을 것 같네요!
> 
> 아하 제가 수정하는 김에 리스너 앱에 맞게 글씨 크기를 맞춰도 될 것 같은데 그건 어떠실까요 ?_?

ㅋㅋㅋㅋㅋㅋㅋ 앗 이미 pr 작성 중입니다...

**Reply by Jsueeee** [reply]
> > > 저희가 '입장을 위한 음성 데이터'처럼 글자 크기를 좀 늘리는 게 나을 것 같네요!
> > 
> > 
> > 아하 제가 수정하는 김에 리스너 앱에 맞게 글씨 크기를 맞춰도 될 것 같은데 그건 어떠실까요 ?_?
> 
> ㅋㅋㅋㅋㅋㅋㅋ 앗 이미 pr 작성 중입니다...

개인적으로 dp를 선호해서 dp로 해놨었는데, 스피커 쪽 sp로 맞췄습니다~

**Reply by Jsueeee** [reply]
![image](https://user-images.githubusercontent.com/66951780/136704876-b72ad8ef-398d-4062-8459-67f85d98e75f.png)

**Reply by jopopcorn** [reply]
> ㅋㅋㅋㅋㅋㅋㅋ 앗 이미 pr 작성 중입니다...

앜ㅋㅋㅋㅋㅋㅋ빠르시네요 알겠습니다~!

**Reply by Jsueeee** [reply]
스피커 레포 보니까 이 부분 텍스트사이즈가 지정이 안된 것 같더라구요! 여기만 신경써주시면 될 것 같습니다!

**Reply by jopopcorn** [reply]
> 스피커 레포 보니까 이 부분 텍스트사이즈가 지정이 안된 것 같더라구요! 여기만 신경써주시면 될 것 같습니다!

알겠습니다 감사해요!!

**Reply by yeonns** [reply]
@Jsueeee @jopopcorn  완료되시면 한번더 비교스샷 부탁드립니당! ❤️ 점점 더 완성도가 높아지고있네요!! 😆 👍

**Reply by Jsueeee** [reply]
> > 혹시 최상단 액션바 색상도 @Jsueeee 님이 올려주신 스샷에서 다른데, 수정이 되었나요? ![image](https://user-images.githubusercontent.com/10149398/136704591-c668c59e-2f1c-4d8d-84c8-6fb6b5e4b951.png)
> 
> 앗 이부분은 수정 안되었습니다!! 상단바 색상 수정해서 올리겠습니다!

```Kotlin
<resources xmlns:tools="http://schemas.android.com/tools">
    <!-- Base application theme. -->
    <style name="Theme.Checkinlistener" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
        <!-- Primary brand color. -->
        <item name="colorPrimary">@color/purple_500</item>
        <item name="colorPrimaryVariant">@color/purple_700</item>
        <item name="colorOnPrimary">@color/white</item>
        <!-- Secondary brand color. -->
        <item name="colorSecondary">@color/teal_200</item>
        <item name="colorSecondaryVariant">@color/teal_700</item>
        <item name="colorOnSecondary">@color/black</item>
        <!-- Status bar color. -->
        <item name="android:statusBarColor">@android:color/transparent</item>
        <item name="windowNoTitle">true</item>
        <item name="android:windowLightStatusBar">true</item>
        <!-- Customize your theme here. -->
    </style>
</resources>
```

저희 theme 코드 공유합니다!

**Reply by Jsueeee** [reply]
> @Jsueeee @jopopcorn 완료되시면 한번더 비교스샷 부탁드립니당! ❤️ 점점 더 완성도가 높아지고있네요!! 😆 👍

넵 😆

**Reply by jopopcorn** [reply]
@Jsueeee @yeonns 
반영한 내용 [여기서](https://github.com/euphony-io/wave-in-speaker/pull/39) 확인 가능합니다!

**Reply by Jsueeee** [reply]
> @Jsueeee @yeonns 반영한 내용 [여기서](https://github.com/euphony-io/check-in-speaker/pull/39) 확인 가능합니다!

수연님 apk 얻을 수 있을까여...❤

**Reply by jopopcorn** [reply]
> 수연님 apk 얻을 수 있을까여...❤

슬랙으로 보내드렸습니다!!

**Reply by yeonns** [reply]
@Jsueeee @jopopcorn 늦은시간까지 너무 고생많으셨습니다!  💯👍 넘나 예쁩니당 ❤️

**Comment by Jsueeee** [comment]
진짜_최종.jpg

<img width="40%" src="https://user-images.githubusercontent.com/66951780/136706159-285aeb3a-7285-4abf-ab07-fa438b066943.png"/> <img width="40%" src="https://user-images.githubusercontent.com/66951780/136706164-879b0d81-60a4-4cc6-91ee-dfe7f2a04207.png"/>

@yeonns @jopopcorn 
비교사진입니다!

**Reply by Jsueeee** [reply]
음...?

**Reply by yeonns** [reply]
음?ㅋㅋ

**Reply by Jsueeee** [reply]
@jopopcorn 혹시 스피커 앱 배경화면 white 설정 안하셨는지 봐주세욬ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by yeonns** [reply]
배경색?

**Reply by jopopcorn** [reply]
앗ㅋㅋㅋㅋㅋ 흰색이 아니었네요..허허허 얼른 같이 올리겠습니다

**Reply by yeonns** [reply]
진짜진짜_최종 이 나오겟네요ㅋㅋ

**Reply by jopopcorn** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 평소에 모든 전자기기에 누런 필터를 끼고 살아서 생각도 못했네요ㅠㅠㅠㅠ

**Reply by Jsueeee** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Comment by Jsueeee** [comment]
@jopopcorn 근데 하나 더 말씀드리자면... 앱 이름도 바꿔야합니다 ^^

**Reply by Jsueeee** [reply]
apk 파일에 안바뀌어있더라구요~ 다신 안주셔도 됩니다!

**Reply by jopopcorn** [reply]
오메.. 이렇게 쫌쫌따리 바꿀게 나오다가 아침이 되어가는거 아니겠죠..!!! 이것도 바꿔서 PR 적용해야겠네요!

**Reply by Jsueeee** [reply]
근데 머라고 하죠...

**Reply by Jsueeee** [reply]
Wave-In

**Reply by Jsueeee** [reply]
wave-in

**Reply by Jsueeee** [reply]
Wave-in

**Reply by jopopcorn** [reply]
음... 대문자 있는게 좋을거같은데 어떡할까요?!
저는 Wave-In이 좋아보이네요!

**Reply by yeonns** [reply]
다른 영어앱이름을 살펴보니 전부 첫글자가 대문자네요ㅋㅋ

**Reply by Jsueeee** [reply]
좋습니다! 저희는 Wave-In (매장용) 이라고 표기할게요!

**Reply by jopopcorn** [reply]
넵넵!!

**Comment by Jsueeee** [comment]
진짜_진짜_최종.jpg

<img width="40%" src="https://user-images.githubusercontent.com/66951780/136706851-1c2d26ef-eaf1-4ff3-bf6e-5a77c78ef7c8.png"/> <img width="40%" src="https://user-images.githubusercontent.com/66951780/136706879-5b4e6656-8509-4fb0-8fa1-2af00e4713d3.png"/>

**Reply by jopopcorn** [reply]
고생하셨습니다~!! 🥰

**Reply by yeonns** [reply]
얼른주무시죠!ㅋㅋ 고생했습니다두분!

**Reply by Jsueeee** [reply]
고생하셨습니다!! :)

**Comment by YoungSeokHong** [comment]
늦은 시간까지 고생 많으셨습니다ㅠㅠ
진짜 세세한 부분까지 신경써주셔서 엄청 완성도가 높아졌네요!
